onnx/defs/math/defs.cc

```diff
@@ -549,3 +549,50 @@ Given two equivalent values, this operator uses the indices along the axis  as
       " axis",
       AttributeProto::INT,
       static_cast<int64_t>(-1));
+
+
+ONNX_OPERATOR_SCHEMA(Unique)
+  .SetDoc(R"DOC(
+Returns the unique scalar elements of the input tensor as a 1-D tensor. The
+order of the elements in the output tensor is unspecified.
+)DOC")
+  .Input(0, "X", "Tensor to find unique elements of", "T")
```

Comment:
I assume X is N dimension and not 1D?

Reply:
@jamesr66a  @bddppq  any update on this?

Reply:
@jamesr66a  Can you take a look?